kbi/0004/index.rst

```diff
@@ -0,0 +1,51 @@
+.. index:: datalad; faq, datalad; annex, datalad git-annex,
+.. highlight:: text
+
+KBI0004: FAQ: is it normal that files are moved to .git/annex/objects when converting existing data into a DataLad dataset?
+===========================================================================================================================
+
+:authors: Stephan Heunis <jsheunis@gmail.com> Christian Mönch <christian.moench@web.de>
+:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/9
+:keywords: datalad faq, git-annex faq, .git/annex/object, symlink, link
+
+This knowledge-base item explains why files are moved into ``.git/annex/objects``, when
+a DataLad dataset is created from an existing dataset, or when a dataset with new files
+is saved.
+
+
+Question:
+---------
+
+When I convert an existing folder to a DataLad dataset, all files are moved to ``./.git/annex/objects/`` and
+the file at the original location becomes a link to the moved content in ``./.git/annex/objects``. Is this
+normal?
+
+
+Answer:
+-------
+
+Yes, this is normal. DataLad manages your data with two main tools: ``git`` and ``git-annex``. The moving and linking is
+used by ``git-annex`` to enable ``git`` to work well with very large files. By applying moving and linking, ``git-annex``
+ensures that ``git`` only needs to track the links, which are quite small in size. The task of transporting and handling
+the data content is performed by ``git-annex``.
+
+This setup creates a modular and portable dataset (the ``git`` repository) which contains information about the versions
+and history of data inside the dataset, while the actual data content is managed by ``git-annex``. File content that is
+placed under management of ``git-annex`` will be moved into ``./.git/annex/objects/`` and a symbolic link (symlink) to
+the content will remain in the original path. This symbolic link will be tracked by ``git``.
+
+The separation of version management (done by ``git``) and content management (done by ``git-annex``)) make a DataLad
+dataset very flexibld. You can for example share the dataset (``git`` repository) publicly, while keeping the contents
+safe on your cluster. People can then access the dataset (with ``datalad clone``) and download individual files in the
```

Comment:
It is probably more accessible to not assume readers know what a "cluster" is in this context, or why one would have one to begin with.